fov_processing_pipeline/tests/test_wrappers.py

```diff
@@ -4,12 +4,19 @@
 
 
 def test_row2im(demo_fov_row):
-    im = wrappers.row2im(demo_fov_row)
+    im, ch_list = wrappers.row2im(demo_fov_row)
 
     assert len(im.shape) == 4
 
+    assert len(ch_list) == 4
 
-def test_save_load_data(demo_cell_data, demo_fov_data, tmpdir, dummy_get_data):
+    assert 'BF' in ch_list
```

Comment:
Should we test to see if the default channel order is correct?

A test to ensure that when ch_order is specified (i.e. wrappers.row2im(demo_fov_row, ["BF"]) returns the correct thing, might be good

a im.shape[0] == len(ch_list) assertion would also be beneficial